stories/demos/HeaderFilters.tsx

```diff
@@ -60,16 +74,17 @@ function createRows() {
 
 export function HeaderFilters() {
   const [rows] = useState(createRows);
-  const [filters, setFilters] = useState<Filters>({
+  const [filters, setFilters] = useState<Filter>({
     task: '',
     priority: 'Critical',
     issueType: 'All',
     developer: '',
-    complete: ''
+    complete: undefined
   });
   const [enableFilterRow, setEnableFilterRow] = useState(true);
 
   const columns = useMemo((): readonly Column<Row>[] => {
+    const headerCellClass = enableFilterRow ? filterContainerClassname : undefined;
```

Comment:
Alternatively we could just set the class on the grid itself, some selectors would need to be tweaked a bit.

Reply:
Done